meta/main.yaml

```diff
@@ -40,5 +40,4 @@ galaxy_info:
     - ssh
 
 dependencies:
-  - role: geerlingguy.repo-epel
-    when: ansible_os_family == "RedHat"
```

Comment:
Why should this dependency be changed? The new one is a fork of the old one and not using Ansible galaxy as origin. The sshguard role does not depend on a specific version of geerlingguy.repo-epel.

If you depend on a fix, please file a pull request at the upstream role.

Reply:
You are completely right. My bad. This is what I get for forgetting to use branches. I did not intend for this pull request to include my temporary edits to the meta dependencies. I will close this pull request and resubmit it with only the commits related to the RHEL vars. I apologize for the confusion. Thanks for your thorough response! It is helping me improve.

Reply:
I will squash the obsolete commits for this pull request .

Reply:
Please review the most recent updates and let me know whether I did it correctly. Thanks!